apps/Tables/pages/InventoryTable.ejs

```diff
@@ -51,12 +51,51 @@
 
     <main role="main" class="container">
 
-      <div class="starter-template">
-      <h1>Welcome</h1>
-      <h3>Something</h3>
-      <h3>Something</h3>
-      <h3>Something</h3>
-      </div>
+        <h2>Inventory Table</h2>
+  
+        <h4>Add New Inventory Item (make this a button)</h4>
+        <a href="/get-data"></a>
+
+        <table>
+          <tr>
+            <th>Item ID</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Length</th>
+            <th>Width</th>
+            <th>Height</th>
+            <th>Scrap?</th>
+            <th>Last Update</th>
+            <th>Employee</th>
+          </tr>
+          <div>
+            {{# each items}}
+              <article class="item">
+                <td>{{this.id}}</td>
+                <td>{{this.type}}</td>
+                <td>{{this.description}}</td>
+                <td>{{this.length}}</td>
+                <td>{{this.width}}</td>
+                <td>{{this.height}}</td>
+                <td>{{this.scrap}}</td>
+                <td>{{this.lastUpdate}}</td>
+                <td>{{this.employee}}</td>
+              </article>
+              {{/each}}
+          </div>
+          <tr>
```

Comment:
I can fill the whole table by manually typing in values, for as many rows as needed... this row prints "Test Item ID", "Test Type" etc.